src/test/java/uk/gov/hmcts/reform/sscs/ccd/deserialisation/SscsCaseCallbackDeserializerTest.java

```diff
@@ -108,4 +104,36 @@ public void should_convert_checked_exception_to_runtime_on_error() throws IOExce
             .hasMessage("Could not deserialize callback")
             .isExactlyInstanceOf(IllegalArgumentException.class);
     }
+
+    @Test
+    public void should_deserialize_appeal_to_proceed_callback() throws IOException {
+        sscsCaseCallbackDeserializer = new SscsCaseCallbackDeserializer(mapper);
+
+        String path = getClass().getClassLoader().getResource("decisionAppealToProceed.json").getFile();
+        String json = FileUtils.readFileToString(new File(path), StandardCharsets.UTF_8.name());
+
+        Callback<SscsCaseData> actualSscsCaseCallback = sscsCaseCallbackDeserializer.deserialize(json);
+
+        SscsInterlocDecisionDocument sscsInterlocDecisionDocument = actualSscsCaseCallback.getCaseDetails().getCaseData().getSscsInterlocDecisionDocument();
+
+        assertEquals("2019-06-26", sscsInterlocDecisionDocument.getDocumentDateAdded());
+        assertEquals("DecisionNotice.pdf", sscsInterlocDecisionDocument.getDocumentFileName());
+        assertEquals("Decision Notice", sscsInterlocDecisionDocument.getDocumentType());
+    }
+
+    @Test
```

Comment:
What about a test for the SscsInterlockDirectionDocument type?

Reply:
I will add it now